### PR TITLE
[Speed] Improves compression docs

### PR DESCRIPTION
**Pull request by angelampcosta**
### Summary

 Improves compression docs.

### Documentation checklist

- [ ] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.

### CODE REVIEW

File: src/content/docs/pages/framework-guides/nextjs/ssr/supported-features.mdx [hunk 1]
```diff
@@ -85,7 +85,7 @@ Cloudflare also supports the older [Pages](https://nextjs.org/docs/pages) router
 - N/A: Not applicable
 ```
 
-[^1]: **compression**: [Cloudflare applies gzip or brotli compression](/speed/optimization/content/brotli/) automatically. When developing locally with Wrangler, no compression is applied.
+[^1]: **compression**: [Cloudflare applies gzip or brotli compression](/speed/optimization/content/compression/) automatically. When developing locally with Wrangler, no compression is applied.
```
Comment (pedrosousa) [inline src/content/docs/pages/framework-guides/nextjs/ssr/supported-features.mdx]:
```suggestion
[^1]: **compression**: [Cloudflare applies Brotli or Gzip compression](/speed/optimization/content/compression/) automatically. When developing locally with Wrangler, no compression is applied.
```
I'd use the same order and capitalization as we have in the compression docs.

File: src/content/docs/speed/optimization/content/compression.mdx [hunk 2]
```diff
@@ -154,3 +154,17 @@ Cloudflare will take into consideration the `accept-encoding` header value in we
...
+
+By default, Cloudflare uses the following compression methods for content delivery, depending on the zone plan. However, the actual compression applied may also depend on what the visitor's browser requests via the `accept-encoding` header.
+
+- Free Plan: Content is compressed by default using Zstandard.
+- Pro and Business Plans: Content is compressed by default using Brotli.
+- Enterprise Plan: Content is compressed by default using Gzip.
+
+### Between Cloudflare and the Origin
+
+On all plans, Cloudflare requests content from the origin server using the `accept-encoding: gzip,br` header. This means that Cloudflare asks the origin to send the content compressed using gzip or Brotli, depending on which method the origin server supports.
```
Comment (pedrosousa) [inline src/content/docs/speed/optimization/content/compression.mdx]:
```suggestion
On all plans, Cloudflare requests content from the origin server using the `accept-encoding: br, gzip` header. This means that Cloudflare asks the origin to send the content compressed using Brotli or Gzip, depending on which method the origin server supports.
```
Matt told me a few days ago that the order is `br, gzip` now.

File: src/content/docs/cache/reference/etag-headers.mdx [hunk 3]
```diff
@@ -30,7 +30,7 @@ When you enable **Respect Strong ETags** in a cache rule, Cloudflare will use st
 However, in some situations Cloudflare will convert strong ETags to weak ETags. For example, given the following conditions:
 
 * **Respect Strong ETags** is enabled
-* [Brotli compression](/speed/optimization/content/brotli/) is enabled
+* [Compression](/speed/optimization/content/compression/) is enabled
```
Comment (angelampcosta) [inline src/content/docs/cache/reference/etag-headers.mdx]:
```suggestion
* [Brotli compression](/speed/optimization/content/compression/) is enabled
```

File: src/content/docs/cache/reference/etag-headers.mdx [hunk 4]
```diff
@@ -60,7 +60,7 @@ When **Respect Strong ETags** is disabled, Cloudflare will preserve strong ETag
 In all other situations, Cloudflare will either convert strong ETag headers to weak ETag headers or remove the strong ETag. For example, given the following conditions:
 
 * **Respect Strong ETags** is disabled
-* [Brotli compression](/speed/optimization/content/brotli/) is enabled
+* [Compression](/speed/optimization/content/compression/) is enabled
```
Comment (angelampcosta) [inline src/content/docs/cache/reference/etag-headers.mdx]:
```suggestion
* [Brotli compression](/speed/optimization/content/compression/) is enabled
```

**Review by ngayerie — APPROVED** [review]

File: src/content/docs/speed/optimization/content/compression.mdx [hunk 5]
```diff
@@ -154,3 +154,17 @@ Cloudflare will take into consideration the `accept-encoding` header value in we
 ---
 
 <Render file="brotli-compression-warning" />
+
+## Compression methods by plan 
+
+### Between Visitors and Cloudflare
```
Comment (pedrosousa) [inline src/content/docs/speed/optimization/content/compression.mdx]:
```suggestion
### Between visitors and Cloudflare
```

File: src/content/docs/speed/optimization/content/compression.mdx [hunk 6]
```diff
@@ -154,3 +154,17 @@ Cloudflare will take into consideration the `accept-encoding` header value in we
...
+
+By default, Cloudflare uses the following compression methods for content delivery, depending on the zone plan. However, the actual compression applied may also depend on what the visitor's browser requests via the `accept-encoding` header.
+
+- Free Plan: Content is compressed by default using Zstandard.
+- Pro and Business Plans: Content is compressed by default using Brotli.
+- Enterprise Plan: Content is compressed by default using Gzip.
+
+### Between Cloudflare and the Origin
```
Comment (pedrosousa) [inline src/content/docs/speed/optimization/content/compression.mdx]:
```suggestion
### Between Cloudflare and the origin server
```

**Review by pedrosousa — APPROVED** [review]

**Pull request merged**